v03_pipeline/lib/reference_data/clinvar.py

```diff
@@ -59,7 +61,7 @@ def safely_move_to_gcs(tmp_file_name, gcs_tmp_file_name):
 
 def parsed_clnsig(ht: hl.Table):
     return (
-        hl.delimit(ht.info.CLNSIG)
+        hl.delimit(ht.CLNSIG)
```

Comment:
These should maybe stay the same?

Reply:
So I was getting this error I didn't quite understand `Error: ExpressionException: Cannot combine expressions from different source objects` and I was able to resolve it by annotating `submitters` and `conditions` to `ht.info` before   parsing the other `selects`. In `parsed_clnsig` and other subsequent accesses to `ht`, the context? of ht is actually  really `ht.info` so I had to remove `info`.

I stopped getting the error after that change.

Reply:
Ah... I think I see what's happening here.   The selects aren't actually evaluated until `get_dataset_ht`... so the annotation that's happening when we call `join_clinvar_to_submission_summary` inside `clinvar_custom_select` is subtly creating a new `Table` under the hood and giving the `different source objects` error.

I think the easiest way out is to just modify the `custom_import` for clinvar to do both the downloading of the vcf and the annotation with the submitters.  Then our "select" becomes super straightforward.  Is that doable?

Reply:
It's doable, and it does also make sense to import everything in the `custom_import` function and not sneak one into the `custom_select`

Reply:
Hope the mocking is also doable!